src/app/view/components/dot-contentlet-editor/components/dot-contentlet-wrapper/dot-contentlet-wrapper.component.ts

```diff
@@ -24,7 +24,10 @@ export class DotContentletWrapperComponent implements OnInit {
     ) {
         if (!this.customEventsHandler) {
             this.customEventsHandler = {
-                'close': (_e: CustomEvent) => {
+                'close': () => {
+                    this.onClose();
+                },
+                'deleted-page': () => {
```

Comment:
don't we close it when the event is 'deleted-contenlet' too?

Reply:
Yes, but from the backend when is not a pageasset we send a `close`